README.md

```diff
@@ -17,25 +17,58 @@
 
 ## Content
 
-1. [Themes](#themes)
-1. [Installation](#installation)
-1. [Overview](#overview)
-1. [Configuration options](#configuration-options)
-   1. [Window](#window)
-   1. [Pane](#pane)
-   1. [Status Line](#status-line)
-   1. [Using the theme's built-in status modules](#using-the-themes-built-in-status-modules)
-1. [Configuration Examples](#configuration-examples)
-   1. [Config 1](#config-1)
-   1. [Config 2](#config-2)
-   1. [Config 3](#config-3)
+<!--toc:start-->
```

Comment:
What is used to generate this toc?

Reply:
https://github.com/jonschlinkert/markdown-toc